### PR TITLE
Fix Editors UserProfile doens't properly filter images. 

### PR DESCRIPTION
**Pull request by tcg**
* Updates Tupos models to utilize optional "partner_id" param, passed to `/api/images` API endpoint. 
* Utilizes that param in the UserProfile component, so that Editor user profiles properly filter the images shown. 

Ticket: https://jira.lifechurch.tv/browse/YV-19027

### CODE REVIEW

**Comment by tcg** [comment]
Related API ticket already deployed in Staging, and working.

**Comment by tcg** [comment]
Looks like CodeClimate is complaining about auto-generated Tupos code.

File: src/tupos/models/partner/index.js [hunk 1]
```diff
@@ -5,166 +5,166 @@ import setString from '@youversion/tupos-base/dist/setters/string'
 import api4 from '@youversion/tupos-base/dist/fetchers/api4'
 
 
-/** *
+/***
  * Partner model
  * @extends TuposModel
  */
 class Partner extends TuposModel {
```
Comment (tcg) [inline src/tupos/models/partner/index.js]: Viewing this PR with whitespace off (add `?w=1` to the URL) shows that only a couple lines here changed at all.

**Comment by michael-martin-al** [comment]
I hid whitespace changes and reviewed this. Looks good to me.

**Pull request merged**